lib/modules/admin-widgets/lib/api.js

```diff
@@ -18,18 +30,27 @@ module.exports = (self, options) => {
   };
 
   self.generateCsv = (ideas) => {
-    const fields = ['titel', 'locatie', 'aangemaakt', 'thema'];
-    const options = { fields };
-
     const mappedIdeas = ideas.map((idea) => {
       return {
         'titel': idea.title,
+        'indiener': idea.user.fullName,
+        'email': idea.user.email,
         'locatie':idea.location && idea.location.coordinates ? idea.location.coordinates[0] + ', ' + idea.location.coordinates[1] : '',
         'aangemaakt': moment(idea.createdAt).format('DD-MM-YYYY HH:mm'),
-        'thema': idea.extraData.theme || ''
+        'thema': idea.extraData.theme || '',
+        'Wat moet er opgeknapt worden?': idea.summary,
+        'Hoe kan dit opgeknapt worden?': idea.description,
+        'Mooie plekken in de wijk': idea.extraData.role,
```

Comment:
Kunnen we deze veldnamen verplaatsen naar de env met een default value die ook voor Amsterdam toepasbaar is? bijv. 'samenvatting', 'beschrijving' en 'eigen rol'. Dan hebben we een iets generiekere oplossing te pakken